powerlibs/django/restless/modelviews.py

```diff
@@ -176,6 +176,10 @@ def patch(self, request, *args, **kwargs):
         if 'PATCH' not in self.methods:
             raise HttpError(405, 'Method Not Allowed')
 
+        for field_name in ['created_by', 'updated_at', 'created_at']:
```

Comment:
Usa uma tupla, aqui, não lista.

Reply:
Ah, outra coisa: tu tá lidando no projeto errado.

E se eu não estiver usando o `OwnedModelMixin` e o campo `created_by` for permitido?

Tem que implementar essas paradas lá no `powerlibs-django-restless-contrib`, nos mixins de endpoints correctos.